downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/BiologicalProcessAnnotationBuilder.java

```diff
@@ -74,8 +73,8 @@ public static void processBiologicalFunctions(GKInstance reactionInst) throws Ex
 
     /**
      * Determines what subunit type this instance has.
-     * @param physicalEntitySchemaClass
-     * @return
+     * @param physicalEntitySchemaClass -- SchemaClass from physicalEntityInst
+     * @return -- String of subunit type that PhysicalEntity has.
```

Comment:
It would be good to indicate the default value if there is no subunit type found.  Maybe this method could be changed to return an empty String by default instead of null?

Reply:
Yea I'll change it to an empty string. It should always return any of the three possible types since it is required to be a Complex/Polymer/EntitySet to reach this function though, so returning the empty string shouldn't be possible